docs/source/examples/checks/distribution/train_test_feature_drift.ipynb

```diff
@@ -3,26 +3,28 @@
   {
```

Comment:
<p>Please add the target=_blank</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>